platform-includes/crons/requirements/elixir.mdx

```diff
@@ -0,0 +1,2 @@
+- Use our <PlatformLink to="/">getting started</PlatformLink> guide to install and configure the Sentry Elixir SDK (min v10.2.0) for your recurring job.
+- [Create and configure](https://sentry.io/crons/create/) your first Monitor.
```

Comment:
```suggestion
- [Create and configure](https://sentry.io/crons/create/) your first monitor.
```
since it's not capitalized in the rest of the doc

Reply:
Ah, fantastique catch 💟